common/os/ami.go

```diff
@@ -0,0 +1,21 @@
+package os
+
+import (
+	"github.com/DataDog/test-infra-definitions/aws"
+	"github.com/pulumi/pulumi-aws/sdk/v5/go/aws/ssm"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+func GetLatestAMI(e aws.Environment, arch Architecture, amd64Path string, armPath string) (string, error) {
```

Comment:
Given it (rightfully) depends on `aws.Environment`. We already have similar functions in `aws/ec2/ec2/ami.go`, so it should probably go there.

Reply:
Code moved.